conf/messages

```diff
@@ -969,10 +969,10 @@ task-list.check-and-send-return.link=Check and send return, pay any tax due
 task-list.upload-supporting-docs.link=Upload Your Supporting Documents
 
 
-task-list.enter-example-property-address.link=Enter details of one property
+task-list.enter-example-property-address.link=Enter details of one company
```

Comment:
the old values still need to be there when it's a multiple disposal journey which isn't indirect disposals

Reply:
oh okay sure, will fix them soon